### PR TITLE
Stem

### PR DESCRIPTION
**Pull request by SekouD**
Added recipe for stem.
https://pypi.org/project/stem/

### CODE REVIEW

**Comment by SekouD** [comment]
Hi,

When I add the test section, the test script complains that the tox.ini file is not found, but it is included with the distribution of the tar.gz file. So I removed the test section.

Should I add it back? If yes how to make sure the test script finds the tox.ini file?

Cheers,

SekouD.

File: recipes/stem/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+{% set hash_type = "sha256" %}
+{% set hash_value = "d7fe1fb13ed5a94d610b5ad77e9f1b3404db0ca0586ded7a34afd323e3b849ed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://files.pythonhosted.org/packages/11/d5/e51983f81b38408ae2f0e166481ad867962f6fa07610fe71119534e12d41/stem-1.6.0.tar.gz
```
Comment (synapticarbors) [inline recipes/stem/meta.yaml]: You could use the canonical url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14

File: recipes/stem/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - tox
```
Comment (synapticarbors) [inline recipes/stem/meta.yaml]: Is `tox` required to build the package?

File: recipes/stem/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - python
+
+test:
+  imports:
+    - stem
+    - stem.descriptor
+    - stem.interpreter
+    - stem.response
+    - stem.util
+  commands:
```
Comment (synapticarbors) [inline recipes/stem/meta.yaml]: Is it necessary to test the existence of `tox` here?

Comment (SekouD) [reply recipes/stem/meta.yaml]: Hi @synapticarbors , how are you?

I made the required changes to the recipe but it seems that the during the test phase of the CI build, when running tox, the file tox.ini is not found but the tox.ini is in the distribution for stem.

Is there a way to point tox to right file?

Thanx.

SekouD.

Comment (synapticarbors) [reply recipes/stem/meta.yaml]: Sorry I missed this comment. I think you'd want:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#source-files

Let me know if you want to give that a try or if I should merge as is.

**Comment by SekouD** [comment]
Hi @synapticarbors , I just saw your comment.
You can merge as is as the tests are passing on the original CI for stem.

Thanks for your feedback.

SekouD.

**Pull request merged**